tests/integration/states/test_file.py

```diff
@@ -3866,3 +3867,54 @@ def test_file_managed_http_source_skip_verify(self):
                              skip_verify=True)
         log.debug('ret = %s', ret)
         self.assertSaltTrueReturn(ret)
+
+
+@destructiveTest
+@skipIf(not salt.utils.is_windows(), 'windows test only')
+class WinFileTest(ModuleCase):
+    '''
+    Test for the file state on Windows
+    '''
+    def setUp(self):
+        self.test = self.run_state('file.managed', name='c:/testfile', makedirs=True, contents='Only a test')
+
+    def tearDown(self):
+        del self.test
```

Comment:
since we are adding a file in the `setUp` we need to make sure are deleting that file in the `tearDown`